app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,39 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.boxed {
+  border: 3px solid green ;
+  width: 60%;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+section {
+  border: 3px solid green ;
+  margin-left: 250px;
+  margin-right: auto;
+  padding: 5px 10px 10px 5px;
+  width: 20%;
+}
+
+.posts {
+  border: 3px solid green ;
+  margin-right: 250px;
+  margin-left: auto;
+  padding: 5px 10px 10px 5px;
+  width: 20%;
+}
```

Comment:
@sharadalt 
For all HTML elements that you'd like to have the green border, consider writing a dedicated style class that can be reused. For example, here is `border-medium` css class that can be reused on any HTML element:

```css
.border-medium {
  border: 3px solid green;
}
```
Then to apply `border-medium` in HTML:

```html
<section class="border-medium">....</section>
<div class="boxed border-medium">...</div>
<p class="posts border-medium">...</div>
```
The above elements will all have the 3px green border.

In addition, this means the `border: 3px solid green;` line can be removed from the CSS at lines 5, 12, and 21 above in `welcome.scss`:

```diff
.boxed {
-  border: 3px solid green;
...
.section {
-  border: 3px solid green;
...
.posts {
-  border: 3px solid green;
```
As always please let me know if you've got questions.